src/CMakeLists.txt

```diff
@@ -40,8 +40,8 @@ function(trilean_option NAME DESCRIPTION DEFAULT_VALUE)
   endif()
 endfunction()
 
-# C++11
-set(CMAKE_CXX_STANDARD 11)
+# C++14
+set(CMAKE_CXX_STANDARD 14)
```

Comment:
if we go with this, please update `INSTALL.md`, at the beginning
```
C/C++ compiler (gcc or visual studio or clang) with C++14 support (i.e. gcc >= 5, clang >= 3.4, msvc >=19).
```

Reply:
@fabiencastan ?

Reply:
Yes, good point. I will fix that.

Reply:
Just checking, I guess it does not affect MeshroomMaya, if i remember correctly the gcc version used to compile maya plugins tended to be very conservative in the past. Dunno if it is still the case, just wanted to remind that in case.

Reply:
For maya 2019
https://around-the-corner.typepad.com/adn/2019/01/maya-2019-api-updates-guide.html
Seems to be fine actually, maybe to check windows because they require Visual Studio 2015 Update 3 that sports msvc 19.00 while the full c++14 comes with later versions. I don't know if cmake check the compiler must fully support the standard or just some of it

for previous versions i guess it depends again on how strict is cmake and if the needed feature was already implemented
https://around-the-corner.typepad.com/adn/2018/02/compiler-versions-for-maya-2018.html